tests/rptest/tests/topic_creation_test.py

```diff
@@ -478,7 +479,15 @@ def __init__(self, test_context):
         self.installer = self.redpanda._installer
         self.rpk = RpkTool(self.redpanda)
 
+    # This test starts the Redpanda service inline (see 'set_up') at the beginning
+    # of the test body. By default, in the Azure CDT env, the service startup
+    # logic attempts to set the azure specific cluster configs.
+    # However, these did not exist prior to v23.1 and the test would fail
+    # before it can be skipped.
     def setUp(self):
+        pass
+
+    def set_up():
```

Comment:
nit: two function names differing only in case/underscore can be a bit hard to read.  Something descriptive like `install_and_start` might be nice.